### DISCUSSION TITLE
post/pagination/

### DISCUSSION THREAD

**Discussion by giscus** [muchlist/paper#13] [General]
# post/pagination/

Membandingkan metode paling optimal dalam penerapan pagination pada backend golang. Implementasi Cursor Based Pagination.

https://blog.muchlis.dev/post/pagination/

**Comment by k1m0ch1** [comment]
anjir,, curhat ini ma ya

**Reply by muchlist** [reply]
wkwk. kalau beda perusahaan masih keliatan curhat gak ya mang ?
